src/picosvg/svg.py

```diff
@@ -986,6 +991,19 @@ def remove_nonsvg_content(self, inplace=False):
 
         return self
 
+    def remove_text(self, inplace=False):
+        if not inplace:
+            svg = self._clone()
+            svg.remove_text(inplace=True)
+            return svg
+
+        self._update_etree()
+
+        for el in self.xpath("//*[name()='text']"):
```

Comment:
I don't understand why SVGs don't allow me to use xpath to find "//text", but according to random bloggers on the web this is the way to do it instead.